lib/app/utils.js

```diff
@@ -1,4 +1,5 @@
 import Vue from 'vue'
+import url from 'urlite/querystring/urlite'
```

Comment:
Can we use `import { parse, format }` for even more optimizations too? :)

Reply:
`'urlite/querystring/urlite'` only exports `parse` and `format`, but for avoiding future changing, I will make the change as your advice.